app/js/components/tabs/tabcomponents/observationComponent.jsx

```diff
@@ -132,32 +132,32 @@ export default class ObsFilter extends React.Component {
   // NM(numeric) datatype
   firstFieldNumeric(){
     return(
-            <div className="form-group" >
-                <label className="col-sm-3 control-label">Which observations?</label>
-                <div className="col-sm-6">
-                    <select 
-                        className="form-control"
-                        value={this.state.timeModifier}
-                        defaultValue="ANY"
-                        name="timeModifier"
```

Comment:
Did you intentionally do all these formatting changes?

Reply:
Yes, I did.  When I  made the pull request, HoundCI pointed out some indentation issues in my files. They occurred after a rebase, to get the changes that had been newly added to  the base branch. I had however missed them while committing, so I went back and fixed them, alongside other styling issues that did not follow eslint best practices.

Reply:
In such cases, we do a separate pull request for just indention or formatting changes, and then another one with the real changes required for this ticket. That way, the reviewer finds it easier to tell what actually changed. So can you do the two separate pull requests?

Reply:
Okay, let me work on doing that. Thank you for letting me know about separating the formatting changes and the feature changes when making a PR, I did not know that beforehand.

Reply:
I have separated the two and created a PR for the the formatting, and I have another branch rebased from that with only the changes concerning the task. Please review the ticket CB-167 for merging, so I can make the PR for the other ticket CB-163 with those changes in place.  Thank you.

Reply:
@LWanjiru i would like to start with the pull request having the actual changes without formatting.

Reply:
Okay.